attendance.php

```diff
@@ -754,21 +754,37 @@ function status_is_changeable_by_user($sta) {
 		display: inline-block;
 		padding: 0.5em 0.75em;
 		background-color: white;
+		border-color: #cecdcd;
+    border-style: solid;
```

Comment:
at some point we should set a consistent tabs/spaces rule and cleanup files to be consistent. i think you're using spaces and the existing file uses tabs. spaces are better in general, but this is about to get ugly real fast.

Reply:
tabs are objectively superior, spaces are ridiculous

but yeah it should just generally match whatever the rest of the file is doing

Reply:
Thanks @adamdruppe I'll be more intentional next time around this. 
![hqdefault](https://user-images.githubusercontent.com/5304109/61651372-9c483700-ac6a-11e9-9a35-a16ce3d730c9.jpg)